app/views/necesidad/necesidades.html

```diff
@@ -2,58 +2,50 @@
       </visualizar-necesidad>
       <div class="panel panel-default" ng-show="necesidades.necesidad.Necesidad.EstadoNecesidadId.Id===1">
-          <div class="panel-body">
-              <div class="row">
-                  <h3 class="text-info">{{"MODALIDAD_SELECCION" | translate}}</h3>
-                  <select ng-show="true" class="form-control"
-                    ng-model="necesidades.modalidadSel"
-                    ng-options="mod as mod.Id+'-'+mod.Nombre for mod in necesidades.modalidad_data track by mod.Id"
-                    required>
-                    <option value="">{{"SELECCIONAR" | translate}} {{"MODALIDAD_SELECCION" | translate}}</option>
-                  </select>
-                </div>
-          
-                
-                <div class="row">
-                  <h3 class="text-info">{{"CLASE_CONTRATACION" | translate}}</h3>
-                  <select ng-show="true" class="form-control" 
-                    ng-options="tc as tc.Id+'-'+tc.TipoContrato for tc in necesidades.tipo_contrato_data track by tc.Id" ng-model="necesidades.TipoContrato"
-                    required>
-                    <option value="">{{"SELECCIONAR" | translate}} {{"TIPO_CONTRATO_NECESIDAD" | translate}}</option>
-                  </select>
-                </div>
+        <div class="panel-body">
+          <div class="row">
+            <h3 class="text-info">{{"MODALIDAD_SELECCION" | translate}}</h3>
+            <select ng-show="true" class="form-control"
+              ng-model="necesidades.modalidadSel"
+              ng-options="mod as mod.Id+'-'+mod.Nombre for mod in necesidades.modalidad_data track by mod.Id"
+              required>
+              <option value="">{{"SELECCIONAR" | translate}} {{"MODALIDAD_SELECCION" | translate}}</option>
+            </select>
+          </div>
+          <div class="row">
+            <h3 class="text-info">{{"CLASE_CONTRATACION" | translate}}</h3>
+            <select ng-show="true" class="form-control"
+              ng-options="tc as tc.Id+'-'+tc.TipoContrato for tc in necesidades.tipo_contrato_data track by tc.Id" ng-model="necesidades.TipoContrato"
+              required>
+              <option value="">{{"SELECCIONAR" | translate}} {{"TIPO_CONTRATO_NECESIDAD" | translate}}</option>
+            </select>
           </div>
-          
+        </div>
       </div>
-
- 
     </div>
 
-
     <div class="modal-footer text-center">
-        <button class="btn btn-success" type="button" name="button" ng-click="necesidades.aprobar_solicitud()" ng-show="necesidades.verBotonAprobarSolicitud">{{"SOLICITAR"
-            | translate}}</button>
+      <button class="btn btn-success" type="button" name="button" ng-click="necesidades.aprobar_solicitud()" ng-show="necesidades.verBotonAprobarSolicitud">{{"SOLICITAR"
+        | translate}}</button>
```

Comment:
De modo similar, por legibilidad al menos lo siguiente:
```suggestion
      <button class="btn btn-success" type="button" name="button"
      ng-click="necesidades.aprobar_solicitud()" ng-show="necesidades.verBotonAprobarSolicitud"
      >{{"SOLICITAR" | translate}}</button>
```
(se puede separar en más lineas)